### PR TITLE
Cindysbranch

### PR DESCRIPTION
**Pull request by cindyxue**
Implementation of Map Part 3.
https://sites.google.com/codeustudents.com/spring-2019/weeks-3-7-guided-projects/maps-part-3?authuser=2

### CODE REVIEW

File: src/main/java/com/google/codeu/data/UserMarker.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class UserMarker {
+
```
Comment (minhphu20) [inline src/main/java/com/google/codeu/data/UserMarker.java]: Extra blank line

File: src/main/java/com/google/codeu/data/UserMarker.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
```
Comment (minhphu20) [inline src/main/java/com/google/codeu/data/UserMarker.java:2]: It is a public class, so would be nice to add a comment on what this class does

**Review by minhphu20 — CHANGES_REQUESTED** [review]

**Review by minhphu20 — APPROVED** [review]

**Comment by minhphu20** [comment]
Could you add in the ReadMe how to use this figure before merging? Thanks

**Comment by cindyxue** [comment]
> Could you add in the ReadMe how to use this figure before merging? Thanks

Is there a format that I could follow for adding info in the ReadMe?

**Pull request merged**